src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -469,7 +469,7 @@ namespace AirflowNetworkBalanceManager {
         if (allocated(ZoneEquipConfig) && NumHybridVentSysAvailMgrs > 0 && allocated(PrimaryAirSystem)) HybridVentilationControl();
         if (VentilationCtrl == 1 && NumHybridVentSysAvailMgrs > 0) AirflowNetworkFanActivated = false;
 
-        if (present(Iter) && present(ResimulateAirZone)) {
+        if (present(Iter) && present(ResimulateAirZone) && SimulateAirflowNetwork >= AirflowNetworkControlSimpleADS) {
```

Comment:
@lgu1234 This avoids the crash, but this section of code still seems wrong for multiple airloops. It seems that AFNSupplyFanType always be zero unless FirstHVACIteration is true, and even the it will be the type for the last airloop only.

Reply:
@mjwitte When FirstHVACIteration is true, it is the current time step. The function is called in SimSelectedEquipment of HVACManager. In addition, AFNSupplyFanType is set by FanTypeNum and we don't change fan type in simulations. We only allow fan mode change for onoff fans. This is taken care of the LoopFanOperationMode. The main purpose is to catch if there is a FanType_SimpleOnOff or not in multiple air loops.

Reply:
OK, I'll trust you on this.